### DISCUSSION TITLE
Add author of upstream change

### DISCUSSION THREAD

**Discussion by MartijnLeijssen-TomTom** [renovatebot/renovate#25261] [Suggest an Idea]
### Tell us more.

When Renovate creates an PR to intake newer versions of our internal components, we would like to automatically invite the author of the integrated (upstream) versions of each component. 

We first discussed if it would be an option to handle reviews based on code owners on the consuming side, but this will give an issue where framework teams will be on every review. 

In our use case we use Github. For our packet manager, the `sourceUrl` property gets properly set for each package, based on this `sourceUrl` property and the old -> new version, the authors of each PR (or commit) could be determined. This functionality should be placed behind a feature configuration flag, and the implementation depends on having `sourceUrl` and the version control system that is being used

What would you think of such feature?

**Comment by MartijnLeijssen-TomTom** [comment]
Could you elaborate why this functionality isn't desired to get in Renovate? 

I could create this functionality outside renovate, but for that it would be really helpful to have access to the data that Renovate produced (packages update, source url, original package version, bumped package version). Is there any API available to retrieve this information?

**Reply by rarkins** [reply]
This sounds like a narrow use case which would add complexity which becomes a maintenance burden. As maintainers of a complex open source package, sometimes we need to say no to more complexity when the benefit is not enough.

Renovate itself is a CLI tools so has no server or API
